wiki/People/Project_Loved_Team/en.md

```diff
@@ -0,0 +1,49 @@
+# Project Loved Team
+
+The **Project Loved Team** are members of the osu! community who manage [Project Loved](/wiki/Project_Loved), the voting platform that promotes beatmaps to the [Loved](/wiki/Beatmaps#loved) category.
+
+## Roles and Responsibilities
+
+The Project Loved Team mostly consists of **captains** for each [game mode](/wiki/Game_Modes). Captains are responsible for nominating beatmaps to the Loved category. They also write short descriptions about the nominated maps to help readers get a feel for each map in case they have not seen or played them before.
+
+The rest of the team helps with quality control, maintenance, and organisation.
+
+## Team Members
```

Comment:
I think this can be divided the same way as pinned on the discord server, per game mode and a part with the rest of the helpers

Reply:
the reason I didn't do this was because I thought it looked weird to have lists for the captains and then a table for the rest (because they have name + role)... should I just do that anyway, idk if it's actually a big concern lol

Reply:
Eh, we can go with this, was just a comment :p